Src/java/tools/cql-formatter/src/main/java/org/cqframework/cql/tools/formatter/Main.java

```diff
@@ -27,11 +27,13 @@ public static void main(String[] args) throws IOException {
         cqlLexer lexer = new cqlLexer(input);
```

Comment:
Can we refactor this, and the corresponding code block from the test cases, to a single static function that takes an InputStream and returns a String that is the formatted output?